test/scripts/integration_tests.py

```diff
@@ -71,6 +73,39 @@ def delete(self, filepath, raise_error=False):
         self.__run_command("delete", filepath, raise_error)
 
 
+class Helm(object):
+    """
+    Class used to interact with helm and return data
+    """
+
+    def delete(self, release_name):
+        """
+        Delete a named helm release
+        """
+        try:
+            subprocess.check_call("helm delete --purge {}".format(release_name), shell=True)
```

Comment:
Does this need shell=True? Usually that is if you need a very specific environment if I remember correctly. Also you can replace this check_call() with run() if you want to skip the error.

Reply:
I'm using shell=True for running locally to include things in my $PATH.